=============== short test summary info =============== 
FIXED app/tests/test_candidate_workflows.py::test_candidate_application_workflow
FIXED app/tests/test_candidate_workflows.py::test_candidate_pipeline_management
FIXED app/tests/test_interviews.py::TestInterviewCreation::test_create_interview_success   
FIXED app/tests/test_interviews.py::TestInterviewCreation::test_create_interview_missing_required_fields
FIXED app/tests/test_interviews.py::TestInterviewCreation::test_create_interview_empty_title
FIXED app/tests/test_interviews.py::TestInterviewCreation::test_create_interview_invalid_type
FAILED app/tests/test_interviews.py::TestInterviewCreation::test_create_interview_unauthorized - NameError: name 'client' is not defined
FIXED app/tests/test_interviews.py::TestInterviewCreation::test_create_interview_insufficient_permissions - (NameError fixed, now 401 auth issue)
FAILED app/tests/test_interviews.py::TestInterviewRetrieval::test_get_interviews_success - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewRetrieval::test_get_interviews_with_filters - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewRetrieval::test_get_interviews_invalid_status_filter - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewRetrieval::test_get_interviews_unauthorized - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewRetrieval::test_get_single_interview_success - AttributeError: <app.services.interview_service.Int...
FAILED app/tests/test_interviews.py::TestInterviewRetrieval::test_get_single_interview_not_found - AttributeError: <app.services.interview_service.Int...
FAILED app/tests/test_interviews.py::TestInterviewUpdate::test_update_interview_success - AttributeError: <app.services.interview_service.Int...
FAILED app/tests/test_interviews.py::TestInterviewUpdate::test_update_interview_empty_title - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewUpdate::test_update_interview_not_found - AttributeError: <app.services.interview_service.Int...
FAILED app/tests/test_interviews.py::TestInterviewUpdate::test_update_interview_unauthorized - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewProposals::test_create_proposal_success - AttributeError: <app.services.interview_service.Int...
FAILED app/tests/test_interviews.py::TestInterviewProposals::test_create_proposal_invalid_time_range - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewProposals::test_respond_to_proposal_accept - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewProposals::test_respond_to_proposal_invalid_response - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewWorkflow::test_cancel_interview_success - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewWorkflow::test_cancel_interview_excessive_reason_length - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewWorkflow::test_reschedule_interview_success - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewWorkflow::test_reschedule_interview_invalid_time_range - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewStats::test_get_interview_stats_success - AttributeError: <app.services.interview_service.Int...
FAILED app/tests/test_interviews.py::TestInterviewStats::test_get_interview_stats_unauthorized - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewCalendarIntegration::test_get_calendar_events_success - AttributeError: <app.services.interview_service.Int...        
FAILED app/tests/test_interviews.py::TestInterviewCalendarIntegration::test_get_calendar_integration_status_success - AttributeError: <app.services.interview_service.Int...
FAILED app/tests/test_interviews.py::TestInterviewEdgeCases::test_create_interview_with_all_optional_fields - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewEdgeCases::test_get_interviews_with_pagination_boundary - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewEdgeCases::test_get_interviews_exceed_limit_boundary - NameError: name 'client' is not defined
FAILED app/tests/test_interviews.py::TestInterviewEdgeCases::test_proposal_with_past_datetime - NameError: name 'client' is not defined
FAILED app/tests/test_interviews_comprehensive.py::TestInterviewEndpoints::test_create_interview_success - assert 403 == 201
FAILED app/tests/test_interviews_comprehensive.py::TestInterviewEndpoints::test_get_interviews_success - pydantic_core._pydantic_core.ValidationError: 44 va...
FAILED app/tests/test_interviews_comprehensive.py::TestInterviewEndpoints::test_get_single_interview_success - AttributeError: <app.services.interview_service.Int...   
FAILED app/tests/test_interviews_comprehensive.py::TestInterviewEndpoints::test_get_single_interview_not_found - AttributeError: <app.services.interview_service.Int... 
FAILED app/tests/test_interviews_comprehensive.py::TestInterviewEndpoints::test_update_interview_success - AttributeError: <app.services.interview_service.Int...       
FAILED app/tests/test_interviews_comprehensive.py::TestInterviewEndpoints::test_create_proposal_success - AttributeError: <app.services.interview_service.Int...        
FAILED app/tests/test_interviews_comprehensive.py::TestInterviewEndpoints::test_respond_to_proposal_accept - AttributeError: 'dict' object has no attribute 'id'        
FAILED app/tests/test_interviews_comprehensive.py::TestInterviewEndpoints::test_cancel_interview_success - assert 403 == 200
FAILED app/tests/test_interviews_comprehensive.py::TestInterviewEndpoints::test_reschedule_interview_success - assert 403 == 200
FAILED app/tests/test_interviews_comprehensive.py::TestInterviewEndpoints::test_get_interview_stats_success - AttributeError: <app.services.interview_service.Int...    
FAILED app/tests/test_interviews_comprehensive.py::TestInterviewEndpoints::test_get_calendar_events_success - AttributeError: <app.services.interview_service.Int...    
FAILED app/tests/test_interviews_comprehensive.py::TestInterviewEndpoints::test_get_calendar_integration_status_success - AttributeError: <app.services.interview_service.Int...
FAILED app/tests/test_interviews_comprehensive.py::TestInterviewEndpoints::test_validation_edge_cases - pydantic_core._pydantic_core.ValidationError: 1 val...
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_create_job_success - fastapi.exceptions.ResponseValidationError: 12 vali...
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_list_jobs_success - pydantic_core._pydantic_core.ValidationError: 30 va...
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_get_job_by_id_success - fastapi.exceptions.ResponseValidationError: 15 vali...
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_get_job_by_slug_success - fastapi.exceptions.ResponseValidationError: 14 vali...
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_update_job_success - fastapi.exceptions.ResponseValidationError: 16 vali...
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_search_jobs_success - TypeError: AsyncClient.get() got an unexpected keyw...
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_get_popular_jobs_success - fastapi.exceptions.ResponseValidationError: 15 vali...
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_get_recent_jobs_success - fastapi.exceptions.ResponseValidationError: 16 vali...
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_get_company_jobs_success - fastapi.exceptions.ResponseValidationError: 16 vali...
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_update_job_status_success - assert 404 == 200
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_bulk_update_status_success - assert 404 == 200
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_update_job_invalid_salary_range - sqlalchemy.exc.IntegrityError: (sqlite3.IntegrityEr...
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_list_jobs_pagination - KeyError: 'skip'
FAILED app/tests/test_jobs.py::TestJobEndpoints::test_bulk_update_empty_job_list - assert 404 in [200, 422]     
FAILED app/tests/test_recruitment_scenario.py::test_basic_recruitment_flow - sqlalchemy.exc.MissingGreenlet: greenlet_spawn has ...
FAILED app/tests/test_recruitment_scenario.py::test_interview_management_workflow - assert 422 == 201
FAILED app/tests/test_recruitment_workflows.py::test_complete_recruitment_workflow - assert 307 == 201
FAILED app/tests/test_recruitment_workflows.py::test_multi_position_recruitment_workflow - assert 307 == 201    
FAILED app/tests/test_recruitment_workflows.py::test_interview_workflow_scenarios - assert 307 == 201
FAILED app/tests/test_todos.py::test_todo_crud_flow - TypeError: can't compare offset-naive and offset-aw...    
FAILED app/tests/test_todos.py::test_overdue_todos_mark_expired - TypeError: can't compare offset-naive and offset-aw...
FAILED app/tests/test_users_management.py::TestUsersManagement::test_reset_password_success - NameError: name 'logger' is not defined
ERROR app/tests/test_jobs.py::TestJobEndpoints::test_create_job_insufficient_permissions
ERROR app/tests/test_jobs.py::TestJobEndpoints::test_get_statistics_forbidden
ERROR app/tests/test_jobs.py::TestJobEndpoints::test_delete_job_forbidden
ERROR app/tests/test_jobs.py::TestJobEndpoints::test_company_jobs_permission_check
= 69 failed, 387 passed, 13 skipped, 6 warnings, 4 errors in 253.66s (0:04:13) =